### ISSUE TITLE
mptcp support no longer in version 0.95

### ISSUE THREAD

**Issue by osumoclement** [multipath-tcp/iproute-mptcp#7]
In version 0.95, there isn't multipath support. Verified by running "ip link help" and there's no multipath option. Version 0.94 has it, and works fine

**Comment by matttbe** [comment]
Hi,

Thank you for this bug report.

Which exact version are you using? `ip -V`
From which package? Can you list all available versions? (maybe the version from your repository is newer than ours)

**Comment by osumoclement** [comment]
Strange. I had tried using version 0.95, compiled it from source, and it didn't have multipath in `ip link help` I was doing that on Manjaro. I even deleted ip binaries from my system, to confirm that my system was now loaded with the binary from `make install`. It still wouldn't work. The version ws `ip utility, iproute2-ss181023`. I compiled version 0.94 `ip utility, iproute2-ss171113` and now this one was working. 

Again, I went to my VPS running Ubuntu, compiled version 0.95. It didn't have multipath. I compiled version 0.94. It had the option.

I don't understand how that "bug" has just disappeared.
